typescript-view-swapping/main.ts

```diff
@@ -0,0 +1,29 @@
+const $TabContainer = document.querySelector('.tab-container');
+const $AllTab = document.querySelectorAll('.tab');
+const $AllView = document.querySelectorAll('.view');
+
+if (!$TabContainer) throw new Error("Didn't work:(");
+
```

Comment:
You should make a more clear error so that we know what "Didn't work:("

Even if it was just "$TabContainer query didn't work:("